internal/transformers/externalmoduleinfo.go

```diff
@@ -47,11 +47,12 @@ func (c *externalModuleInfoCollector) collect() *externalModuleInfo {
 	hasImportDefault := false
 	for _, node := range c.sourceFile.Statements.Nodes {
 		switch node.Kind {
-		case ast.KindImportDeclaration:
+		case ast.KindImportDeclaration, ast.KindJSImportDeclaration:
```

Comment:
Is this necessary? Aren't these imports type-only and just elided?

Reply:
Yes, they are type-only. In general KindJS* should only affect .d.ts emit since they're generated from real statements in .js files.